src/apps/lwaftr/nh_fwd.lua

```diff
@@ -582,39 +584,74 @@ local function test_ipv6_flow ()
    test_ipv6_service_to_vm({pkt1})
 end
 
-local function test_ipv4_cache_trigger ()
-   local pkt = packet.from_string(lib.hexundump([[
+local function test_ipv4_cache_trigger (pkt)
+   local ether_dhost = "52:54:00:00:00:01"
+   local refresh_packet = ipv4_cache_trigger(pkt, ethernet:pton(ether_dhost))
+   local eth_hdr = ethernet:new_from_mem(refresh_packet.data, ethernet_header_size)
+   local ip_hdr = ipv4:new_from_mem(refresh_packet.data + ethernet_header_size, pkt.length - ethernet_header_size)
+   assert(ip_hdr:src_eq(n_cache_src_ipv4))
+   assert(ethernet:ntop(eth_hdr:dst()) == ether_dhost)
+   lwdebug.print_pkt(refresh_packet)
```

Comment:
The task is still WIP (notice the [WIP] tag in the subject). I will remove the line when it's ready to merge. Thanks for noticing it anyway.